msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/brokerapi/TestCase1561087.java

```diff
@@ -44,8 +44,28 @@ public void test_1561087() {
         mBroker.setFlights(flightsJson);
         Assert.assertEquals(flightsJson, mBroker.getFlights());
 
+        // Add flights and get to check if the flight information is returned
+        final String anotherFlightJson = "{\"AnotherFlight\":\"hello\"}";
+        mBroker.addFlights(anotherFlightJson);
+        Assert.assertEquals( "{\"AnotherFlight\":\"hello\",\"SetFlightsTest\":\"true\"}", mBroker.getFlights());
+
+        // Override flights and get to check if the flight information is returned
+        final String flightToOverwrite = "{\"SetFlightsTest\":\"false\"}";
+        mBroker.addFlights(flightToOverwrite);
+        Assert.assertEquals("{\"AnotherFlight\":\"hello\",\"SetFlightsTest\":\"false\"}", mBroker.getFlights());
+
+        // Add flight with null value. the flight map should not change.
+        final String flightMapWithNullValue = "{\"SetFlightsTest\": null}";
```

Comment:
what does setting null means?
should it remove the flight/reset to default value?
if we want to no-op it, should we return an error/ throw exception?

Reply:
modified. Setting null means we'll remove the flag - if it exists.